OpenRA.Game/Game.cs

```diff
@@ -190,6 +191,10 @@ public static void InitializeSettings(Arguments args)
 
 		internal static void Initialize(Arguments args)
 		{
+			foreach (var f in Launch.GetType().GetFields())
```

Comment:
This shouldn't be mixed in with the Settings code - launch args aren't settings.